code/gui/serialbox.h

```diff
@@ -33,11 +34,31 @@ Q_OBJECT
 
     Q_SLOT void append_text(const std::string &text);
 
+    Q_SLOT void up_slider_changed(int value);
+
+    Q_SLOT void down_slider_changed(int value);
+
+    Q_SLOT void left_slider_changed(int value);
+
+    Q_SLOT void right_slider_changed(int value);
+
+    Q_SLOT void up_box_changed();
+
+    Q_SLOT void down_box_changed();
+
+    Q_SLOT void left_box_changed();
+
+    Q_SLOT void right_box_changed();
+
+    void slider_changed(int value, int dir);
+
+    void box_changed(int value, int dir);
+
 private:
     enum Power {
-        POWER_INTERVAL = 8,
+        POWER_INTERVAL = 1,
         POWER_MAX = 255,
-        POWER_MIN = 0
+        POWER_MIN = 240
```

Comment:
If these don't need to be seen in the header put them in the source file

Reply:
Both enums?